homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -34,15 +39,27 @@
         vol.All(cv.ensure_list, [vol.In(list(SENSORS))])})
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, add_entities,
```

Comment:
Please rename `add_entities` to `async_add_entities`.
```suggestion
async def async_setup_platform(hass, config, async_add_entities,
```